test-e2e/common/esy-prefix-via-esyrc.test.js

```diff
@@ -10,11 +11,11 @@ it('Common - esy prefix via esyrc', async () => {
   expect.assertions(2);
   const p = await initFixture(path.join(__dirname, './fixtures/simple-project'));
 
-  await del('/tmp/custom-esy-prefix', {force: true});
+  await del(path.join(os.homedir(), '.esytest', 'custom-esy-prefix'), {force: true});
```

Comment:
Using `homedir` because the `tmpdir` was too long on my mac and made test fails because of it

Reply:
Nice catch - that seems reasonable, as long as we clean it up at the end.

Reply:
Everything that is not `~/.esytest/esy` is cleaned up after all the tests are done so that should be fine